docs/GSoC.md

```diff
@@ -0,0 +1,80 @@
+
+It’s a huge count of the number of users and organizations. 
+
+There have been 1.0 billion commits in last year, along with 25.3 million active repositories, 12.5 million active issues and 47 million public and private pull requests merged.
+
+Also, since September 2016, around 6.7 million developers have joined Github, 451 thousand new organizations have been created with around 1.2 million people making their first pull request and 191 thousand people staring or forking a repository without pushing code.
+
+These figures are really huge and we can conclude that GitHub has been an alternative to the word Open Source. We can also conclude from the above data that a lot of activities happen on GitHub.
+
+Many of these activities, which are really significant to the users contributing in the organization or a repository goes unnoticed and currently, there is no way by which maintainers can generate a report and summarize details of the numbers of public commits, active issues, pull requests or comments on issues and other useful data.
+
+There are many organizations which generate a weekly report of all the events occurring in their organizations. However, their weekly reports or weekly digests are manually generated or are generated by external bots, and these bots are only organization depended and they need to be configured per organization or per-repository basis. Also, these external bots are not First Actors in GitHub.
+
+It can be seen that users or organizations are highly interested in maintaining a weekly digest for their repository, however, due to the lack of an application, which does this task, they have to do it manually or using some external application.
+
+## The Solution
+
+To solve this problem, I developed the Weekly Digest app based on Probot Framework which generates a weekly automated summary of activity on your GitHub repository.
+
+Th app listens to [Github Webhook Events](https://developer.github.com/webhooks/#events) like `installation.created` and `installation_repositories`, and also to `schedule.repository` which are the GitHub events that trigger the application.
+
+The app then reads the configuration of the application (if it's available) and checks the details that it has to publish using [probot-config](https://www.npmjs.com/package/probot-config).
+
+Then the application gets details like issues, pull requests, commits, contributors, stargazers and releases for the repositories (Using [Octokit REST APIs](https://octokit.github.io/rest.js/)) and adds [GitHub Markdown](https://guides.github.com/features/mastering-markdown/) to it.
+
+Finally, the application publishes the fetched details an issue and adds _weekly-digest_ label to it. This is repeated every week, by default on Sunday.
+
+### So, what the app does?
```

Comment:
I think that is not grammatically correct? Maybe change it to "So, what is the app doing?"